types/reaction.d.ts

```diff
@@ -0,0 +1 @@
+declare module "@reactioncommerce/components/ShopLogo/v1";
```

Comment:
Do we need this?

Reply:
This prevents a type error using modules from the component library. This will eventually become a list of items as they are used from the lib, each added when it becomes used. Not the most straight-forward thing but to me sane until there's a better approach (I haven't found one yet).

<img width="811" alt="Screen Shot 2021-04-01 at 6 46 02 PM" src="https://user-images.githubusercontent.com/65664227/113283267-aa44cb00-931a-11eb-9166-f698d8777495.png">

Reply:
Ok yes I was also expecting this to become a larger list at some point and therefore wondered if we couldn't avoid that somehow. But IMHO not a blocker, something that could easily be refactored later on.